sdk/contentsafety/Azure.AI.ContentSafety/api/Azure.AI.netstandard2.0.cs

```diff
@@ -0,0 +1,17 @@
+namespace Azure.AI
```

Comment:
You need to have a service area under the topic. See https://azure.github.io/azure-sdk/general_design.html#general-namespaces-naming for guidelines.

Reply:
@heaths I think this problem is related to the bug I encountered when generating sdk project following this [guidance](https://github.com/Azure/azure-sdk-for-net/blob/main/doc/DataPlaneCodeGeneration/AzureSDKPackage_Setup.md)
What this step produced all have the name Azure.AI instead of Azure.AI.ContentSafety. I manually fixed most of the names and missed this one.
I'm wondering what's your suggestion here? Since running eng/scripts/CodeChecks.ps1 contentsafety would not fix this problem. Do I need to fix manually? Or do you know how to fix this issue in script?
You can reproduce it by running: 
.\eng\scripts\automation\Invoke-TypeSpecDataPlaneGenerateSDKPackage.ps1  -sdkFolder D:\src\mengaims\azure-sdk-for-net\sdk\contentsafety\Azure.AI.ContentSafety -typespecSpecDirectory specification\cognitiveservices\ContentSafety -commit 462e8d9327ea0dfc77e7df76933b34b3d489953e -repo Azure\azure-rest-api-specs-pr
And I pushed the output sdk project [here](https://github.com/mengaims/azure-sdk-for-net/tree/acs/sdk/contentsafety).

Reply:
I've done some manual fix and ran eng/scripts/CodeChecks.ps1 contentsafety to update. Could you check if the new file Azure.AI.ContentSafety.netstandard2.0.cs is valid for you?

Reply:
Fixed namespace manually